clippy_lints/src/copies.rs

```diff
@@ -360,3 +364,36 @@ where
 
     None
 }
+
+fn search_same_list<T, Hash, Eq>(exprs: &[T], hash: Hash, eq: Eq) -> Option<Vec<(&T, &T)>>
```

Comment:
Why return an Option here? Is there any more information than you get if the Vec is empty?

Reply:
I did not think of this. I'll make the proper arrangements.